actions/run.py

```diff
@@ -27,6 +27,6 @@ def run(self, **kwargs):
 
         try:
             api_response = getattr(api_instance, sdk_method)(**kwargs)
-            pprint(api_response)
+            return jsons.dump(api_response)
```

Comment:
Can you explain or confirm what this does? Based on what you said, I am under the impression that returning this JSON object makes it available to an action as the "result.result" object?

Reply:
yep see the example i posted in the comments

so result contains the dict then LM API returns the response from the API as a result object for some objects

Reply:
We could try api_response.result but some things will break :(

Reply:
Yeah, the built-in StackStorm feature is that data returned from the actions/sensors in json format can be automatically used as a structured object in the workflows and rules.

Reply:
Ok, good to know. We will definitely be including this change in a future release and I'm very happy that you brought it to my attention. I'll take a look at the above rule and action so that I can confirm this in internal testing. Hopefully you guys will stay available if I have any questions. Appreciate you two sharing your expertise.